src/plugins/columnSorting/columnSorting.js

```diff
@@ -654,7 +654,7 @@ class ColumnSorting extends BasePlugin {
       return;
     }
 
-    const physicalColumn = this.hot.toPhysicalColumn(column);
+    const physicalColumn = this.hot.renderedToPhysicalColumn(column);
```

Comment:
We've agreed that `afterGetColHeader` will not be called for hidden columns and we can keep the argument to be `visible` index, not `rendered` one.